tests/modules/core/lib/Auth/Process/PHPTest.php

```diff
@@ -113,10 +112,8 @@ public function testThrowExceptionFromFilter()
             ]
         ];
 
-        $this->expectException(
-            \Exception::class,
-            "Missing uid attribute."
-        );
+        $this->expectException(Exception::class);
+        $this->expectExceptionMessage("Missing uid attribute.");
         $this->processFilter($config, $request);
```

Comment:
Not sure in what PHPunit-version this was allowed, but today it's split in two calls (actually for years already)